test/functional/apps/console/_autocomplete.ts

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default function ({ getService, getPageObjects }: FtrProviderContext) {
+  const retry = getService('retry');
+  const log = getService('log');
+  const PageObjects = getPageObjects(['common', 'console']);
+
+  describe('console autocomplete feature', function describeIndexTests() {
+    this.tags('includeFirefox');
+    before(async () => {
+      log.debug('navigateTo console');
+      await PageObjects.common.navigateToApp('console');
+      // Ensure that the text area can be interacted with
+      await PageObjects.console.dismissTutorial();
+    });
+
+    it('should provide basic auto-complete functionality', async () => {
+      expect(await PageObjects.console.hasAutocompleter()).to.be(false);
+      await PageObjects.console.enterRequest();
+      await PageObjects.console.promptAutocomplete();
+      await retry.waitFor('autocomplete to be visible', () =>
+        PageObjects.console.hasAutocompleter()
```

Comment:
Right now this function is lying about the result of its work. It is checking if the autocomplete is present as an element. But at the same time, it can have style `display: none`.